v0/util/index.js

```diff
@@ -174,6 +180,14 @@ const formatTimeStamp = (dateStr, format) => {
       return date.getTime();
   }
 };
+
+// Accepts a timestamp and returns the corresponding unix timestamp
```

Comment:
I think your case is already handled with the case: `secondTimestamp` in the `handleMetadataForValue` method. WDYT?

Reply:
Yes its already handled with that case. Thanks for pointing there. Using secondTimeStamp now